SocketIOClientSwift/SocketTypes.swift

```diff
@@ -27,8 +27,9 @@ import Foundation
 public typealias AckCallback = ([AnyObject]) -> Void
 public typealias NormalCallback = ([AnyObject], SocketAckEmitter?) -> Void
 public typealias OnAckCallback = (timeoutAfter: UInt64, callback: AckCallback) -> Void
+public typealias SocketOptionsSet = Set<SocketIOClientOption>
```

Comment:
I would remove the typealias and use ```Set<SocketIOClientOption>``` directly in the API because everybody knows about Set but nobody knows about this secret SocketOptionsSet